src/components/IconPicker/IconDisplay.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { Box, Typography } from "@mui/material";
+import {
+  Home,
+  Settings,
+  Person,
+  Search,
+  Add,
+  Edit,
```

Comment:
imho, the icons here seems to be a bit weird given the specificity of the project, I'll create a ticket to maybe consider alternative list ;)

Reply:
yes it's just a list random